SS2-Project/Assets/Starstorm2/Components/MimicInventoryManager.cs

```diff
@@ -0,0 +1,172 @@
+﻿using UnityEngine;
+using RoR2;
+using System.Collections.Generic;
+using System.Collections;
+using UnityEngine.AddressableAssets;
+
+namespace SS2.Components
+{
+    [RequireComponent(typeof(CharacterBody))]
+    [RequireComponent(typeof(PurchaseInteraction))]
+    public class MimicInventoryManager : MonoBehaviour
+    {
+        private List<ItemIndex> itemInd = new List<ItemIndex>();
+        public Transform pivot;
```

Comment:
Add a comment explaining what this pivot is for or give it a better variable name